README.md

```diff
@@ -89,7 +89,7 @@ flatpak run hr.from.josipantolis.starfish
 
 ## License
 
-[GNU GPLv3](COPYING)
+[GNU GPLv3](LICENSE)
```

Comment:
I think COPYING is [the standard / recommended](https://www.gnu.org/licenses/gpl-howto.html) name for the file containing the full GPL license text.
I'd like to stay aligned with that standard.